app/services/return-requirements/check-your-answers.service.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+/**
+ * Orchestrates fetching and presenting the data for `/return-requirements/{sessionId}/check-your-answers` page
+ * @module CheckYourAnswersService
+ *
+ * @param {string} id - The UUID for return requirement setup session record
+ *
+ * @returns {Promise<Object>} page data needed by the view template
+ */
+const CheckYourAnswersPresenter = require('../../presenters/return-requirements/check-your-answers.presenter.js')
+const SessionModel = require('../../models/session.model.js')
+
+async function go (sessionId) {
```

Comment:
Just some tidying up at the top of the module

```suggestion
'use strict'

/**
 * Orchestrates fetching and presenting the data for `/return-requirements/{sessionId}/check-your-answers` page
 * @module CheckYourAnswersService
 */

const CheckYourAnswersPresenter = require('../../presenters/return-requirements/check-your-answers.presenter.js')
const SessionModel = require('../../models/session.model.js')

/**
 * Orchestrates fetching and presenting the data for `/return-requirements/{sessionId}/check-your-answers` page
 *
 * @param {string} sessionId - The UUID for return requirement setup session record
 *
 * @returns {Promise<Object>} page data needed by the view template
 */
async function go (sessionId) {
```